patchwork/steps/ExtractCode/ExtractCode.py

```diff
@@ -30,6 +30,8 @@ def get_source_code_context(
         context = "".join(source_lines[position.start : position.end])
         if count_openai_tokens(context) <= context_token_length:
             return position.start, position.end
+        else:
+            logger.warn("The selected context is larger than the context size limit. Try increasing the context_size.")
```

Comment:
Just a nitpick but can you make sure to use `context_size` in both places instead of context size in the first line to be consistent. Also may be useful to print the actual limit. So, something like `The selected context is larger than the contex_size limit of {context_token_length}. You can increase the context_size and try again.`

Reply:
Made the suggested changes.